docs/Compile.rst

```diff
@@ -51,15 +71,19 @@ Dependencies
 ------------
 DFHack is meant to be installed into an existing DF folder, so get one ready.
 
-We assume that any Linux platform will have ``git`` available.
+We assume that any Linux platform will have ``git`` available (though it may
+require installing from your package manager.)
 
-To build DFHack you need a version of GCC 4.x capable of compiling for 32-bit
+To build DFHack you need GCC version 4.5 or later, capable of compiling for 32-bit
 (i386) targets. GCC 4.5 is easiest to work with due to avoiding libstdc++ issues
-(see below), but any later 4.x version should work as well. GCC 5.x will not
-work due to ABI changes (the entire plugin loading system won't work, for
-example). On 64-bit distributions, you'll need the multilib development tools
-and libraries (``gcc-multilib`` or ``gcc-4.x-multilib`` on Debian). Note that
-installing a 32-bit GCC on 64-bit systems (e.g. ``gcc:i386`` on Debian) will
+(see below), but any version from 4.5 onwards (including 5.x) will work.
+
+On 64-bit distributions, you'll need the multilib development tools and libraries:
+
+* ``gcc-multilib`` and ``g++-multilib``
+* On Debian: ``gcc-4.x-multilib`` and ``g++-4.x-multilib``
```

Comment:
OK, so just remove those Debian lines?  That's not anything I've edited, just brought over from the original version of the doc (with updated formatting.)